tools/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualAppInfo.scala

```diff
@@ -136,6 +136,20 @@ class QualAppInfo(
     probNotDataset.values.flatten.toSet.mkString(":")
   }
 
+  private def getAllPotentialProblems(getPotentialProb: String, nestedComplex: String): String = {
+    val nestedComplexT = if (nestedComplex.nonEmpty) "NESTED COMPLEX TYPE" else ""
```

Comment:
so we don't print the actual type here then?  Please add documentation to this function and explain.